package.json

```diff
@@ -95,7 +95,15 @@
     "build:app": "REACT_APP_GIT_SHA=$VERCEL_GIT_COMMIT_SHA react-scripts build",
     "build:version": "node ./scripts/build-version.js",
     "build:prebuild": "node ./scripts/prebuild.js",
+    "build:packages": "yarn build:pkg:excalidraw && yarn build:pkg:utils",
+    "build:packages:dev": "yarn build:pkg:excalidraw:dev",
+    "build:pkg:excalidraw": "yarn --cwd src/packages/excalidraw build:umd",
+    "build:pkg:excalidraw:dev": "yarn --cwd src/packages/excalidraw build:umd:dev",
+    "build:pkg:utils": "yarn --cwd src/packages/utils build:umd",
```

Comment:
Any specific reason for adding these commands to root package.json ?
In near future we most likely will move to `monorepo` structure with `yarn workspace` which means every package will have its own relevant files and deps so these commands will not be present in root file, rather it will be present in its own `package.json`

Reply:
Done in 777a5ff.